src/sidebar/components/tag-editor.js

```diff
@@ -192,6 +192,15 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
         }
         e.preventDefault();
         break;
+      case 'Tab':
+        if (activeItem === -1) {
+          // allow tab key to add a suggestion but only if
+          // it matches what was typed
```

Comment:
What's "it" here? Does this mean: "Allow `tab` key to add the `input` value as a tag, but only if the `input` value is a full match for a suggestion?"

Reply:
ya, that comment was bad, how about this?
```
// allow tab key to add tag, but only if the <input> value
// matches a suggestion from the list
```